include/fmt/core.h

```diff
@@ -73,7 +73,8 @@
 # endif
 #endif
 
-#if FMT_HAS_FEATURE(cxx_explicit_conversions)
+#if FMT_HAS_FEATURE(cxx_explicit_conversions) || \
+    FMT_MSC_VER >= 1800
```

Comment:
MSVC 18.0 is the minimum supported version in `master`, so the check is not needed.

Reply:
Since MSVC doesn't support __has_feature, surely there needs to be some check? FMT_MSC_VER > 0 seems like a silly test.

Reply:
My bad, not sure what I was thinking about =).